include/zephyr/arch/arm64/hypercall.h

```diff
@@ -15,5 +15,7 @@ int HYPERVISOR_hvm_op(int op, void *param);
 int HYPERVISOR_memory_op(int op, void *param);
 int HYPERVISOR_grant_table_op(int op, void *uop, unsigned int count);
 int HYPERVISOR_domctl(void *param);
+int HYPERVISOR_xen_version(int op, void *param);
+int HYPERVISOR_sysctl(void *param);
```

Comment:
Please, cover this with `CONFIG_XEN_INITIAL_DOMAIN`, we missed the same for domctl, it will be added in `zephyr-v3.3.0-xt` branch.